config/app.default.php

```diff
@@ -178,6 +182,7 @@
             'password' => 'secret',
             'client' => null,
             'tls' => null,
+            'url' => env('EMAILTRANSPORT_DEFAULT_URL', ''),
```

Comment:
To be consistent with underscore usage elsewhere, shouldn't this be `EMAIL_TRANSPORT_DEFAULT_URL`?